index-pt-br.html

```diff
@@ -98,28 +98,28 @@
 </p>
 
 <p>
-  Pessoas que patrocinam também estão sujeitas às políticas antiassédio. Em particular, elas 
+  Representantes de empresas patrocinadoras também se aplicam às políticas antiassédio. Em particular, 
```

Comment:
I think this would be "A representantes de empresas patrocinadoras,  também se aplicam as políticas antiassédio.

Reply:
Agreed.

Reply:
(but without the comma :)  )